src/mixins/bcol-mixin.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, Vue } from 'vue-property-decorator'
+import axios from '@/axios-auth'
+
+@Component({})
+export default class BcolMixin extends Vue {
+  async getErrorObj (errCode): Promise<any> {
```

Comment:
The methods seem pretty straight forward so i'm not sure if you absolutely NEED doc bloc comments, but i find in a mixin for example, where the sole purpose is to export these methods to use elsewhere, it never hurts :)

Reply:
Severin wants them, so in they go!

Reply:
@cameron-freshworks I wanted the comments to explain the `.catch()` (so it didn't look like an oversight).